### ISSUE TITLE
Teach merge conflicts as bonus content

### ISSUE THREAD

**Issue by kelly-sovacool** [UMSWC/curriculum#84]
Sometimes learners ask about it, although there isn't typically enough time to teach it in the main lesson.

**Comment by zenalapp** [comment]
Good idea - this way we can also direct them to the curriculum if they as and we don't have time to teach it.
